__tests__/workers/brief/userGenerateBrief.ts

```diff
@@ -0,0 +1,109 @@
+import { expectSuccessfulTypedBackground, saveFixtures } from '../../helpers';
+import { userGenerateBriefWorker as worker } from '../../../src/workers/brief/userGenerateBrief';
+import { DataSource } from 'typeorm';
+import createOrGetConnection from '../../../src/db';
+import { Source, User } from '../../../src/entity';
+
+import { usersFixture } from '../../fixture/user';
+import { typedWorkers } from '../../../src/workers';
+import { BriefingModel, BriefingType } from '../../../src/integrations/feed';
+import { BriefPost } from '../../../src/entity/posts/BriefPost';
+import { sourcesFixture } from '../../fixture';
+import { generateShortId } from '../../../src/ids';
+import nock from 'nock';
+
+let con: DataSource;
+
+beforeAll(async () => {
+  con = await createOrGetConnection();
+});
+
+describe('userGenerateBrief worker', () => {
```

Comment:
Just a note, this will now include more stuff from the sources/posts?

Reply:
Davide is adding those statistics I will then add them as flags.

Reply:
You want to wait for re-review once that's added?